components/chainhook-sdk/src/indexer/stacks/mod.rs

```diff
@@ -35,6 +35,36 @@ pub struct NewBlock {
     pub transactions: Vec<NewTransaction>,
     pub events: Vec<NewEvent>,
     pub matured_miner_rewards: Vec<MaturedMinerReward>,
+
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub block_time: Option<u64>,
+    
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub signer_bitvec: Option<String>,
+    
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub signer_signature: Option<Vec<String>>,
+
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub cycle_number: Option<u64>,
+
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub reward_set: Option<RewardSet>,
```

Comment:
stacks-core isn't very consistent with some of these fields being optional (undefined) vs nullable. So goal here is to just support either case during deserialization.